shelley/chain-and-ledger/executable-spec/test/Generator/Update/QuickCheck.hs

```diff
@@ -152,15 +152,14 @@ genRho = genIntervalInThousands 1 9
 genTau :: Gen UnitInterval
 genTau = genIntervalInThousands 100 300
 
--- | activeSlotCoeff: 0-1
+-- | activeSlotCoeff: 0.1-1
```

Comment:
should we restrict this to non-zero in the spec, too ? not sure about the step-size, though

Reply:
I think so, yes. `f=0` is completely degenerate.